test/langtools/tools/lib/toolbox/JavacTask.java

```diff
@@ -331,6 +333,29 @@ public String name() {
         return checkExit(new Task.Result(toolBox, this, rc, outputMap));
     }
 
+    /**
+     * Sets the expected outcome of the task and calls {@code run(procs)}.
+     * @param expect the expected outcome
+     * @return the result of calling {@code run}
+     */
+    public Result run(Expect expect, Processor... procs) {
```

Comment:
I would prefer to add a new method to this class, something in the lines of:
```
public JavacTask processors(Processor... procs) {
    this.procs = List.of(procs);
    return this;
}
```

which I think is more in the spirit of the existing API, then you won't need to add the two new versions of the `run` method

Reply:
Added as suggested; thanks.